icons/svg/newsstand.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 -960 960 960"><path d="M80-160v-60h800v60H80Zm89-160v-316h60v316h-60Zm159 0v-480h60v480h-60Zm159 0v-480h60v480h-60Zm275 0L597-600l54-30 162 280-51 30Z"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 -960 960 960" width="24px" fill="#5f6368"><path d="M80-160v-80h800v80H80Zm80-160v-320h80v320h-80Zm160 0v-480h80v480h-80Zm160 0v-480h80v480h-80Zm280 0L600-600l70-40 160 280-70 40Z"/></svg>
```

I think the SVG that you can get from [here](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:newsstand:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=newsstand&icon.size=24&icon.color=%235f6368&icon.platform=web&icon.set=Material+Symbols) is a better match for the other paragon icons. It's the one showing as "Newstand2" below:

![Screenshot 2024-11-08 at 10 45 46 AM](https://github.com/user-attachments/assets/c3f645dd-0908-43cf-a040-02cd906dd100)

It's also the correct size, 24x24, not 48x48.

Reply:
@sdaitzman any preference ?